src/openzaak/components/documenten/migrations/0027_auto_20230417_1415.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2022 Dimpact
+# Generated by Django 3.2.18 on 2023-04-17 14:15
+
+from django.db import migrations
+
+
+def populate_private_voltooid_attribute(apps, schema_editor):
+    BestandsDeel = apps.get_model("documenten", "BestandsDeel")
+
+    for bestandsdeel in BestandsDeel.objects.all():
+        bestandsdeel._voltooid = bestandsdeel.inhoud.size == bestandsdeel.omvang
```

Comment:
Will this cause a call over CMIS for every document in a migration (when CMIS is used)?

In that case, I much rather evaluatie and save this when its consulted.

Reply:
I don't think so, `BestandsDeel` is just a regular Django model that does not use CMIS (from what I can see)